app/views/main.scala.html

```diff
@@ -67,47 +75,34 @@
 
     @inlineScript = {
 
-    @scriptElement
-
-    @if(configDecorator.analyticsToken == Some("N/A")) {
-        <script type="text/javascript">
-                var ga = function (send, event, category, action, label, args) {
-                    console.log(JSON.stringify({
-                        call: 'ga',
-                        send: send,
-                        event: event,
-                        category: category,
-                        action: action,
-                        label: label
-                    }));
-
-                    if (args && typeof args !== 'undefined' && typeof args['hitCallback'] === 'function') {
-                        var hitCallback = args['hitCallback'];
-                        hitCallback();
-                    }
-
-                };
-                window._gaq.push = function (args) {
-                    console.log(JSON.stringify({
-                        call: '_gaq.push',
-                        args: args
-                    }));
-                };
-        </script>
-    }
 
-    @if(displayTimeoutDialog) {
-        <script>
+@scriptElement
+
+@if(configDecorator.analyticsToken == Some("N/A")) {
+    }
+<script type="text/javascript">
+                    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+                    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+                    ga('create', '@configDecorator.analyticsToken', '@configDecorator.analyticsHost');
+                    ga('set', 'anonymizeIp', true);
+@gaCustomEvent
+                    </script>
+
+<script>
                 window.GOVUK.timeoutDialog({
                     timeout: @configDecorator.sessionTimeoutInSeconds,
                     countdown: @configDecorator.sessionCountdownInSeconds,
                     keepAliveUrl: '@controllers.routes.SessionManagementController.keepAlive()',
                     signOutUrl: '@controllers.routes.SessionManagementController.timeOut()',
                 });
         </script>
+@if(displayTimeoutDialog) {
```

Comment:
This should be wrapped around:

`            window.GOVUK.timeoutDialog({	                window.GOVUK.timeoutDialog({
                    timeout: @configDecorator.sessionTimeoutInSeconds,	                    timeout: @configDecorator.sessionTimeoutInSeconds,
                    countdown: @configDecorator.sessionCountdownInSeconds,	                    countdown: @configDecorator.sessionCountdownInSeconds,
                    keepAliveUrl: '@controllers.routes.SessionManagementController.keepAlive()',	                    keepAliveUrl: '@controllers.routes.SessionManagementController.keepAlive()',
                    signOutUrl: '@controllers.routes.SessionManagementController.timeOut()',	                    signOutUrl: '@controllers.routes.SessionManagementController.timeOut()',
                });`

Reply:
done